samples/Gallery/Pages/NotificationsPage.fs

```diff
@@ -73,13 +114,15 @@ module NotificationsPage =
 
             model, []
 
+        //TODO What is this about?
```

Comment:
WindowNotificationManager can't be used immediately after creating it, so we need to wait for it to be attached to the visual tree. See https://github.com/AvaloniaUI/Avalonia/issues/5442

Reply:
I've converted that into a comment.